agent/build.gradle.kts

```diff
@@ -138,7 +138,7 @@ tasks {
             developer {
               id.set("splunk")
               name.set("Splunk Instrumentation Authors")
-              email.set("support+sonatype@signalfx.com")
+              email.set("support+java@signalfx.com")
```

Comment:
is this a thing? Who get these emails, anyway?

Reply:
No idea 🤷‍♂️ 
`support+java` was what was in the signalfx-java repo, so I wanted to use the same value. And I used it as the GPG key name too.

Reply:
I'll put in a quick PR to change Android to use this as well

Reply:
👍 

I'm doing the same thing for other repos too